docs/cpp/cpp-ide.md

```diff
@@ -49,74 +51,105 @@ ColumnLimit: 0
+Doxygen is a tool that generates documentation from source code. When you annotate code with comments, Doxygen will then generate documentation for these functions. For doxygen comments, type `/**` and press `Enter` to generate a doxygen comment block. Supported doxygen tags include: `@brief`,`@tparam`, `@param`, `@return`, `@exception`, `@deprecated`, `@note`, `@attention`, and `@pre`.
 
-With the peek window open, you browse the list of competing definitions to find the one you're interested in. If you want to navigate to the location of one of the definitions just double-click the definition you're interested in, or by double-clicking anywhere in the source code displayed on the left-hand side of the peek window.
+### Markdown comments
 
-## Navigate source code
+The C++ Extension by default supports showing a subset of markdown in your editor. This subset supports all markdown comments except the symbols "_" and "*". Toggle the new **Markdown in Comments** setting to enable all markdown, keep this subset of markdown, or disable markdown support.
 
-The source code navigation features provided by the C/C++ extension are powerful tools for understanding and getting around in your codebase. These features are powered by tags stored in a local database of symbol information. With the C/C++ extension installed, this database is generated whenever a folder containing C++ source code files is loaded into VS Code. The database icon appears next to the active configuration name ("Win32" in the image below) while the tag-parser is generating this information.
+## Navigating source code
 
-![The platform indicator during tag parsing](images/cpp/parsing.png)
+The source code navigation features provided are powerful tools for understanding for your codebase. These let you quickly search for symbols in your code, navigate to their definitions, and find references to them.
 
-The icon disappears when all the symbols have been tagged.
+Navigation is powered by a set of tags stored in a local database of symbol information. Whenever a folder containing C++ source code files is opened, the  C/C++ extension creates a database of the symbols defined in those files. This database is updated whenever a file is saved, and shared between all workspaces that use the same source code.
 
 ### Search for symbols
 
 You can search for symbols in the current file or workspace to navigate your code more quickly.
 
-To search for a symbol in the current file, press `kb(workbench.action.gotoSymbol)`, then enter the name of the symbol you're looking for. A list of potential matches will appear; it is filtered as you type. Choose from the list of matches to navigate to its location.
+To search for a symbol in the current file, press `kb(workbench.action.gotoSymbol)`, then enter the name of the symbol you're looking for. A list of potential matches appears; which is filtered as you type. Choose from the list of matches to navigate to its location.
```

Comment:
This `;` should be `,` in order to be grammatically correct. The clause after the semicolon needs to be "independent", but it's not after the changed text.